fighters/ganon/src/status/special_n_float.rs

```diff
@@ -93,26 +94,17 @@ unsafe extern "C" fn special_n_float_main_loop(fighter: &mut L2CFighterCommon) -
     // Only perform these actions if vars::ganon::status::FLOAT_ENABLE_ACTIONS is true.
     if VarModule::is_flag(fighter.battle_object, vars::ganon::status::FLOAT_ENABLE_ACTIONS) {
         // if the proper transition terms are enabled, these functions will check for
-        // if Ganon performs an aerial or a double jump.
+        // if Ganon performs an aerial, airdodge, or a double jump.
         if fighter.sub_transition_group_check_air_cliff().get_bool()
         || fighter.sub_transition_group_check_air_attack().get_bool()
-        || fighter.sub_transition_group_check_air_jump_aerial().get_bool() {
+        || fighter.sub_transition_group_check_air_jump_aerial().get_bool()
+        || fighter.sub_transition_group_check_air_escape().get_bool() {
             return 1.into();
         }
         // If Special is pressed, enable a flag and transition into the next status.
         if fighter.global_table[globals::PAD_FLAG].get_i32() & *FIGHTER_PAD_FLAG_SPECIAL_TRIGGER != 0
         || fighter.global_table[globals::STICK_Y].get_f32() <= -0.7 {
-            VarModule::on_flag(fighter.battle_object, vars::ganon::status::FLOAT_CANCEL);
```

Comment:
instead of removing the ~~very high effort~~ cancel animation, you can probably get the same effect by adding the following before the return statement:
```
fighter.clear_lua_stack();
lua_args!(fighter, FIGHTER_KINETIC_ENERGY_ID_GRAVITY);
let speed_y = sv_kinetic_energy::get_speed_y(fighter.lua_state_agent);
sv_kinetic_energy!(
    reset_energy,
    fighter,
    FIGHTER_KINETIC_ENERGY_ID_GRAVITY,
    ENERGY_GRAVITY_RESET_TYPE_GRAVITY,
    0.0,
    speed_y,
    0.0,
    0.0,
    0.0
);

Reply:
Didn't work. Still falls like a brick